exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -413,4 +413,8 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  String USE_DYNAMIC_UDFS = "exec.udf.use_dynamic";
```

Comment:
I did not use the existing option "exec.udf.enable_dynamic_support" because if that option is enabled and then disabled later, expectation is that dynamic UDFs added in that window continue to be available and working even after disabling.  

All other comments addressed. Please review new diffs.